### DISCUSSION TITLE
Is it possible to use execution options for other Jupyter kernels?

### DISCUSSION THREAD

**Discussion by fsmunoz** [quarto-dev/quarto-cli#8541] [Q&A] [answered]
### Description

I'm using the SAS kernel for JupyterLab, and things are working mostly as expected; one detail that I would like to improve is removing the output from some cells, in this case the first:

![image](https://github.com/quarto-dev/quarto-cli/assets/285727/c4ca93af-b174-413e-b648-b1104dbee0c2)

SAS is on the verbose side by default, and while that can be configured, it raised the topic of execution options: I see that there are [execution options](https://quarto.org/docs/computations/execution-options.html) to control various aspect of the execution, but even though it's documented that other Jupyter kernels are supported (and this is true, since this works), the execution options seem to assume that `#` is the comment character, which is not the case for SAS (`* foo ;`  or `/* foo */` is).

Is there something I can do to make execution options work in this situation?

## Environment

* Quarto 1.4.549
* JupyterLab 3.6.7
* Debian GNU/Linux 13 "trixie"
* Emacs 30.0.5 with `quarto-mode` (ESS for SAS blocks):
![image](https://github.com/quarto-dev/quarto-cli/assets/285727/b52a676a-7a92-47cc-ab91-a538a2b95c5d)

**Comment by mcanouil** [comment, marked as answer]
It does not assume the comment symbol is the same, it uses as default `#` if none is known.

- https://github.com/quarto-dev/quarto-cli/blob/abd34d34072947b95be723a8087eef762bb9fe9e/src/core/jupyter/jupyter.ts#L1134-L1180

**Comment by fsmunoz** [comment]
Thank you @mcanouil , that's great to know, and you've put me into the right direction: I was ommiting the `|`, this works as expected:
```{sas}
*| output: false;
```
![image](https://github.com/quarto-dev/quarto-cli/assets/285727/733df239-6baa-425e-9692-f354df9a6f9f)
